features/networking/multus.feature

```diff
@@ -1385,6 +1385,16 @@ Feature: Multus-CNI related scenarios
       | ["metadata"]["name"]      | macvlan-bridge-25657    |
       | ["metadata"]["namespace"] | <%= project(-1).name %> |
     Then the step should succeed
+    # Check NAD is configured under project namespace
```

Comment:
@weliang1 try below:
`And admin checks that the "macvlan-bridge-25657" network_attachment_definition exists in the "<%= project(-1).name %>" project`

Reply:
Update it with suggestion.
New test log: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/6033/console